doc/Publishing.md

```diff
@@ -142,3 +142,10 @@ Updating the `yarn.lock` helps identify potential problems with our dependency v
 To successfully complete a `yarn upgrade`, one must:
 - perform `yarn upgrade` at the root of the repository.
 - fix any compilation errors, typing issues, and failing tests that may be introduced.
+
+### Update Milestones
+
+* Close current release milestone.
```

Comment:
We can make this a link to the actual [milestones](https://github.com/eclipse-theia/theia/milestones) to make it easier for others to follow if necessary.

Reply:
done